src/main/java/com/icthh/xm/uaa/web/rest/UserResource.java

```diff
@@ -233,6 +233,23 @@ public ResponseEntity createUser(@Valid @RequestBody UserDTO user) throws URISyn
         return new ResponseEntity<>(page.getContent(), headers, HttpStatus.OK);
     }
 
+    /**
+     * GET  /users/by/keys : get users by keys.
+     *
+     * @param pageable the pagination information
+     * @return the ResponseEntity with status 200 (OK) and with body of users
+     */
+    @GetMapping("/users/by/keys")
+    @Timed
+    public ResponseEntity<List<UserDTO>> getUsersByKeys(
```

Comment:
Why not to use Page<UserDTO>

Reply:
please elaborate

Reply:
do u mean use Page<UserDTO> instead of List<> ?
i have copied it from getAllUsers(...) method. Do u think using Page is preferable ?

Reply:
Sorry but can i get any feedback here ?